spec/features/user_signs_in_spec.rb

```diff
@@ -5,25 +5,23 @@
   I want to sign in
   So that I can regain access to my account
 } do
-  # scenario 'specify valid credentials', js: true do
-    # user = FactoryGirl.create(:user)
+  scenario 'specify valid credentials' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.